### DISCUSSION TITLE
How can I find details about the architecture of the default CNN in Spacy?

### DISCUSSION THREAD

**Discussion by nsmoore** [explosion/spaCy#13593] [Help: Model Advice]
Is there documentation or an API for retrieving the various layers of the CNN and how they are connected? We are trying to understand how the Spacy CNN model works for text classification at a more fine-grained level.

**Comment by ecly** [comment]
@nsmoore in case it's helpful, then you can see how the different architectures are composed in the model source code where they are registered. E.g. for textcat:  https://github.com/explosion/spaCy/blob/master/spacy/ml/models/textcat.py
